webserver/views/user.py

```diff
@@ -98,6 +103,31 @@ def import_data():
                            lastfm_username=lastfm_username)
 
 
+@user_bp.route("/export")
+@login_required
+def export_data():
+    filename = request.args.get("filename")
```

Comment:
At least for firefox, i just have a "Save As" and "Open With" option. Mostly depends on user's configuration, i think.